app/modules/synology/client/DownloadApi.scala

```diff
@@ -0,0 +1,53 @@
+package modules.synology.client
+
+import com.google.inject.Inject
+import modules.core.auth.models.LoginStatus
+import modules.synology.models.downloads.{DownloadStatus, Downloads}
+import play.api.libs.json.{JsError, JsSuccess, Json, Reads}
+import play.api.libs.ws.WSClient
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class DownloadApi @Inject() (wsClient : WSClient) (implicit ec: ExecutionContext) {
```

Comment:
Looks okay, but the api prefix should be done with a config... 

also download api and client api should be one